src/ts/constants/networks.ts

```diff
@@ -0,0 +1,27 @@
+import { Chain, Polkadot, Substrate } from './chains'
+
+export type Network = {
+  chain: Chain,
+  name: string,
+  rpcServer: string,
+  txExplorer: string
+}
+
+export const Alexander: Network = {
+  chain: Polkadot,
+  name: 'Alexander',
+  rpcServer: 'wss://poc3-rpc.polkadot.io/',
+  txExplorer: 'https://polkadot.js.org/apps/#/explorer'
+}
+
+export const CharredCherry: Network = {
+  chain: Substrate,
+  name: 'Charred Cherry',
+  rpcServer: 'wss://substrate-rpc.parity.io/',
+  txExplorer: 'https://polkadot.js.org/apps/#/explorer'
+}
+
+export const chains: {[chainType: string]: Array<Network>} = {
```

Comment:
maybe create a type for `{[chainType: string]: Array<Network>}`

Reply:
I didn't declare the type here as I don't think the type will be referenced. We can explicitly declare the type if we need it in future probably

Reply:
yeah. np